.github/ISSUE_TEMPLATE/suggest_wallet.md

```diff
@@ -6,98 +6,192 @@ labels: "wallet :purse:,"
 
 **Is your wallet globally accessible?**
 
 <!-- Please list any KYC requirements or geographic limitations of your wallet, if any exist. -->
 
-**Is your wallet custodial, non-custodial, or a hardware wallet?**
+**Is there internationalization support for your wallet?**
+
+<!-- Please list languages that the wallet actively supports -->
+
+**What is the repository for your codebase?**
+
+<!-- If you’re project is open source, please provide a link to the codebase. -->
+
+**What social links are there for the project?**
+
+<!-- Please provide social links for the wallet (Discord, Twitter, etc.) -->
+
+**What is your wallet's onboarding experience?**
+
+<!-- Please provide an explanation of where to find the user onboarding experience (links, is it built into the app, etc.), and what the onboarding experience is. -->
+
+### Device
+
+**Does the wallet have a mobile app? If yes, which operating systems are supported (iOS, Android)?**
+
+<!-- If the wallet has a mobile interface, please provide information and links to the app store for the operating systems supported. -->
+
+**Does the wallet have a desktop app? If yes, which operating systems are supported (Windows, Mac, Linux)?**
+
+<!-- If the wallet has a desktop interface, please provide links and information for the operating systems supported. -->
+
+**Does the wallet have a browser extension? If yes, which browsers are supported (Chromium, Firefox, Safari)?**
+
+<!-- If the wallet has a browser extension, please provide links and information for the browsers that are supported. -->
+
+**Is it a hardware wallet?**
+
+<!-- How is it used / how does it connect (e.g. USB)? What software wallets does it integrate with (e.g. MetaMask)? -->
+
+### Security
+
+**Is the wallet code open-sourced?**
+
+<!-- If yes, please provide a direct link to the repository. -->
+
+**What license was the wallet software released under?**
+
+<!-- Please provide information on the software license used for the wallet. -->
+
+**Is your wallet custodial, or non-custodial?**
 
 <!-- Do users have access to their public and/or private keys? If your company/project were to disappear, would users still be able to access their funds? -->
 
 **Please describe the measures taken to ensure the wallet's security and provide documentation wherever possible**
 
-<!-- Please provide a link to a report or repo. If you haven't been audited but think your wallet should be listed anyway, explain here. -->
+<!-- Please provide a link to any security audits or code reports. If you haven't been audited but think your wallet should be listed anyway, explain here. -->
 
-**Does the wallet support layer 2 networks?**
+**Spam protection?**
 
-<!-- Please list which L2 solutions the wallet supports. -->
+<!-- Does the wallet employ any practices to warn users against potential spam (e.g. when interacting with suspicious accounts/contracts)? -->
```

Comment:
Is spam the right word here? I'm curious do we know which wallets have features like this?